lib/data/queries/flight2019/getFlights2019.ts

```diff
@@ -1,5 +1,13 @@
 import prisma from "../../../../prisma/client";
 
-export default async function getFlights2019() {
-  return await prisma.flight2019.findMany();
+export default async function getFlights2019(...args: string[]) {
+  return args.length > 0
+    ? await prisma.flight2019.findMany({
+        where: {
+          departmentId: {
+            equals: args[0],
+          },
+        },
+      })
+    : await prisma.flight2019.findMany();
```

Comment:
It is not clear to me why we use an array of strings if we only ever use its first element. Furthermore I would refactor – not sure though whether defining such filter objects is good practice for prisma – this function as follows.
```suggestion
export default async function getFlights2019(departmentId?: string) {
  const filter = departmentId
    ? {
        departmentId: {
          equals: departmentId,
        },
      }
    : undefined;

  return await prisma.flight2019.findMany({
    where: filter,
  });
```

Does this makes sense to you @Pei-Nie? If you have an idea on how to do it differently. Please let me know.
Please make sure